src/app/cars/cars.component.ts

```diff
@@ -7,9 +7,18 @@ import { Component, OnInit } from '@angular/core';
 })
 export class CarsComponent implements OnInit {
 
-  constructor() { }
+  constructor() {}
 
   ngOnInit() {
   }
 
+  toggleTable(name: string) {
```

Comment:
There are better ways of approaching the hiding/showing of divs in angular, such as the [`ngIf` directive](https://angular.io/api/common/NgIf).

In your component:
```
export class myClass {
show: boolean = true;
}
```
In your html:
```
<div *ngIf="show">Content to render when condition is true.</div>
<button (click)="show = !show">click me</button>
```